spec/controllers/affiliates_controller_spec.rb

```diff
@@ -5,7 +5,7 @@
   render_views
 
   before(:all) { create_users }
-
+  before(:each) { @authable = create(:facility) }
```

Comment:
Where did this become necessary? Doesn't look like you're touching this controller at all.